UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.Common/Configuration/D365PayloadKeyConfiguration.cs

```diff
@@ -0,0 +1,12 @@
+﻿
+namespace UKHO.ExternalNotificationService.Common.Configuration
```

Comment:
It seems to me like the D365 payload should be fixed, not configured. Is there are a reason why.

Reply:
PostEntityImages key is being discussed in progress right now, all config changes will be removed once the discussion is completed.

Reply:
once  property name should be consistent.